public/index.html

```diff
@@ -3,14 +3,23 @@
 <link rel='stylesheet' href='/style.css'>
 
 <h1>devPunks \m/</h1>
-<h2>The first technology show from junior to super senior developer in 15 minutes.</h2>
-
-<blockqoute>
-</blockquote>
+<h2>the first technology show from <marquee style='color:red'>?????</marquee> to super developer in 15 minutes.</h2>
```

Comment:
@melaniegershman suggestions for the `???` please ;-) You know better than I do.